### PR TITLE
TELCODOCS-1453: Adding SNO post-install config for OLM

### PR DESCRIPTION
**Pull request by rohennes**
[TELCODOCS-1453](https://issues.redhat.com//browse/TELCODOCS-1453): SNO post-install recommended config to remove OLM monitoring component.

Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/TELCODOCS-1453

Link to docs preview:
https://68117--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu#ztp-sno-du-reducing-resource-usage-with-olm-pprof_sno-configure-for-vdu

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mpmaruthu** [comment]
Telco CNF QE: Looks good to me.
/Peri M

File: modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-du-reducing-resource-usage-with-olm-pprof_{context}"]
+= Operator Lifecycle Manager
+
+{sno-caps} clusters that run DU workloads require consistent access to CPU resources. Operator Lifecycle Manager (OLM) periodically gathers performance data from Operators, which causes a spike in CPU usage. The following `ConfigMap` custom resource (CR) disables the gathering of Operator performance data by OLM.
```
Comment (agantony) [inline modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc:9]:
```suggestion
{sno-caps} clusters that run DU workloads require consistent access to CPU resources. The Operator Lifecycle Manager (OLM) collects performance data from operators at regular intervals, resulting in an increase in CPU utilisation. The following `ConfigMap` custom resource (CR) disables the collection of Operator performance data by the OLM.
```

File: modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+.Recommended cluster OLM configuration (`ReduceOLMFootprint.yaml`)
```
Comment (agantony) [inline modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc:11]: (`ReduceOLMFootprint.yaml`) is this required?

File: modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc (hunk 1 above)
Comment (agantony) [reply modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc:9]: DU ??

File: snippets/ztp_ReduceOLMFootprint.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: collect-profiles-config
+  namespace: openshift-operator-lifecycle-manager
+data:
+  pprof-config.yaml: |
+    disabled: True
+
```
Comment (agantony) [inline snippets/ztp_ReduceOLMFootprint.yaml:9]: extra space??

**Review by agantony — COMMENTED** [review]
Good job. Added a few suggestions for your consideration; overall lgtm!

File: snippets/ztp_ReduceOLMFootprint.yaml (hunk 3 above)
Comment (rohennes) [reply snippets/ztp_ReduceOLMFootprint.yaml:9]: I think it's generally best practice to leave a space at the end of a file. Snippets perhaps are a bit different but doesn't really make much difference in the output

File: modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc (hunk 1 above)
Comment (rohennes) [reply modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc:9]: Great rewrite, thanks! FYI - OLM should have "the" before it according to OCP [glossary](https://github.com/openshift/openshift-docs/blob/30dc46d020d38222dea3277536910424be756775/contributing_to_docs/term_glossary.adoc#operator-lifecycle-manager-olm).

File: modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc (hunk 2 above)
Comment (rohennes) [reply modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc:11]: Yes - it links to a file in a recommend cluster spec container

**Pull request merged**